console-extensions.json

```diff
@@ -9,5 +9,45 @@
         "kind": "FlashSystemCluster"
       }
     }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "ibmstoragesystem",
+      "insertBefore": "persistentvolumes",
+      "section": "storage",
+      "name": "IBM Flash Systems",
+      "href": "/k8s/all-namespaces/odf.ibm.com~v1alpha1~FlashSystemCluster"
+    },
+    "flags": {
+      "required": ["ODF_IBM_STORAGE"]
+    }
+  },
+  {
+    "type": "console.page/resource/details",
+    "properties": {
+      "model": {
+        "group": "odf.ibm.com",
+        "version": "v1alpha1",
+        "kind": "FlashSystemCluster"
+      },
+      "component":{"$codeRef": "IBMStorageODFDetailPage"}
+    },
+    "flags": {
+      "required": ["ODF_IBM_STORAGE"]
+    }
+  },
```

Comment:
```suggestion
```

Reply:
this will be removed when odf-console is ready to deploy